app/build.gradle.kts

```diff
@@ -56,29 +56,43 @@ android {
     composeOptions {
         kotlinCompilerExtensionVersion = "1.5.3"
     }
+
+    flavorDimensions.add("type")
+    productFlavors {
+        create("Api") {
```

Comment:
I think it is better to use small cases for the flavor names

Reply:
Ok